.github/workflows/test-sui.yaml

```diff
@@ -0,0 +1,149 @@
+name: Test Sui
+
+on: pull_request
+
+env:
+  SUI_VERSION: mainnet-v1.25.3
```

Comment:
let's define this version in the repo, say somewhere like `sui/version.json`. Then, we can read it here, and also within `sui/deploy-contract.js` and print a warning if a different CLI version is being used then what has been tested.

Reply:
how about storing this in the [GitHub Variables Settings](https://github.com/axelarnetwork/axelar-contract-deployments/settings/secrets/actions) instead of reading from file to simplify it?